src/main/java/Duke.java

```diff
@@ -1,10 +1,247 @@
+import java.time.format.DateTimeParseException;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.stream.Stream;
+
 public class Duke {
+    private ArrayList<Task> db = new ArrayList<>(10);
```

Comment:
Perhaps a more intuitive variable name here?

Reply:
Personally, I thought `db` was a very intuitive name since it commonly refers to 'database'. A good point to consider though, thank you!